### PR TITLE
Minor edits to Metadata and Lineage Documentation

### PR DESCRIPTION
**Pull request by awholegunch**
Passes [Quick Build 1](http://builds.cask.co/browse/CDAP-DOB108-1)

### CODE REVIEW

File: cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst [hunk 1]
```diff
@@ -23,8 +23,8 @@ CDAP captures metadata from many different sources |---| as well as those specif
 user |---| on different entities and objects. The container model of CDAP provides for the
 seamless aggregation of a wide variety of machine-generated metadata that is automatically
 associated with datasets. This capability allows flexibility for the developers and data
-scientist to innovate and build solutions on Hadoop without any responsibility for
-compliance and governance.
+scientist to innovate and build solutions on Hadoop, while simultaneously having a lineage
```
Comment (bdmogal) [inline cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst]: scientists?

Comment (bdmogal) [inline cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst]: 'having a lineage' does not sound right. perhaps 'without having to worry about maintaining compliance and governance in every application'

File: cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst [hunk 2]
```diff
@@ -23,8 +23,8 @@ CDAP captures metadata from many different sources |---| as well as those specif
 user |---| on different entities and objects. The container model of CDAP provides for the
 seamless aggregation of a wide variety of machine-generated metadata that is automatically
 associated with datasets. This capability allows flexibility for the developers and data
```
Comment (bdmogal) [inline cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst]: remove 'the'?

Also, should we remove the word 'capability'? This sentence sounds like an alliteration.

File: cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst]: Fixed.

**Comment by bdmogal** [comment]
LGTM

**Pull request merged**